### ISSUE TITLE
Can not change learning in fine-tuning!

### ISSUE THREAD

**Issue by Coderx7** [Microsoft/caffe#111]
Hello, I noticed When I'd train my network using a configuration such as below: 

```
base_lr: 0.1
momentum: 0.9
weight_decay:  0.001
type:"AdaDelta"
delta:1e-3
gamma: 0.1  
lr_policy: "multistep"
stepvalue: 50000
stepvalue: 95000
stepvalue: 253000
```

 And then try to fine-tune it using snapshots belonging to the second or third step, the learning rate can not be changed, meaning in the second step, the learning rate becomes 0.01 and in the third it becomes 0.001. now If for fine-tuning I use a snapshot belonging to the second step, and doesnot change the steps but the base learning rate for the fine-tuning process, no matter what I do, I can not decrease the learning rate. It simply gets stuck with the solver settings it was initially trained!
I have to change the step values at least to make it use the new solver settings.!
